cmake/FindCoCoA.cmake

```diff
@@ -53,6 +53,15 @@ if(NOT CoCoA_FOUND_SYSTEM)
 
   get_target_property(GMP_LIBRARY GMP IMPORTED_LOCATION)
 
+  find_package(Patch)
```

Comment:
Can we make this REQUIRED and remove the message below? We usually don't give examples on how to install a package on different platforms.

Reply:
Or we can do `message(FATAL_ERROR "Can not build CoCoA, missing binary for patch")` similar to what is done in FindCLN.cmake

Reply:
Thanks for the feedback!

I've been giving some thought to the problem, and I had the idea to install the binary for 'patch' so that the building process of CoCoA can use it. This would make things seamless for the user. The approach would be something like this:

https://github.com/alanctprado/cvc5/blob/dd7e235331a19805360c8317522f114a81958929/cmake/FindCoCoA.cmake#L56-L70

If this is acceptable, I'll clean up the code and squash the commits. Otherwise, I can change it to throw the message you've mentioned :)

Reply:
I don't think this is necessary. We don't need to go crazy with dependencies. Let's just report back the missing dependency that the user has to install. `patch` is a pretty standard tool.